api/PaymentMethodChangeEvent.json

```diff
@@ -15,12 +15,36 @@
             "version_added": "79"
           },
           "firefox": {
-            "version_added": false,
-            "notes": "Available only in nightly builds. Requires <code>dom.payments.request.enabled</code> to be set to <code>true</code> and the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA."
+            "version_added": "55",
+            "notes": "Available only in nightly builds.",
```

Comment:
The note is still here :)

Reply:
Ahh, I misread your comment before, my bad!